package.json

```
@@ -77,52 +82,55 @@
     "@types/htmlbars-inline-precompile": "^3.0.0",
     "@types/qunit": "^2.11.2",
     "@types/rsvp": "^4.0.4",
-    "@typescript-eslint/eslint-plugin": "^4.32.0",
-    "@typescript-eslint/parser": "^4.32.0",
-    "babel-eslint": "^10.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.57.1",
+    "@typescript-eslint/parser": "^5.57.1",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^2.2.0",
-    "ember-cli": "~3.28.1",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^6.0.0",
+    "concurrently": "^8.0.1",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli": "~4.12.3",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
-    "ember-qunit": "^6.0.0",
-    "ember-resolver": "^8.0.3",
+    "ember-qunit": "^6.2.0",
+    "ember-resolver": "^10.0.0",
     "ember-sinon": "^5.0.0",
     "ember-sinon-qunit": "^6.0.0",
-    "ember-source": "~3.28.0",
+    "ember-source": "~4.12.0",
     "ember-source-channel-url": "^3.0.0",
     "ember-try": "^2.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^11.0.6",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.0.0",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-ember": "^11.5.0",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.4",
     "husky": ">=6",
     "lint-staged": ">=10",
     "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.3.2",
-    "qunit": "^2.16.0",
+    "prettier": "^2.8.7",
+    "qunit": "^2.19.4",
     "qunit-sinon-assertions": "^1.0.0",
     "release-it": "^15.4.1",
     "release-it-lerna-changelog": "^5.0.0",
+    "stylelint": "^15.4.0",
+    "stylelint-config-standard": "^32.0.0",
+    "stylelint-prettier": "^3.0.0",
     "typedoc": "^0.23.14",
     "typescript": "^4.4.3",
-    "webpack": "5"
+    "webpack": "^5.78.0"
   },
   "resolutions": {
     "@babel/plugin-transform-modules-amd": "7.16.5"
   },
+  "peerDependencies": {
+    "ember-source": ">= 3.28.0"
+  },
   "engines": {
-    "node": "18.* || 20.* || >= 22"
+    "node": "14.* || 16.* || >= 18"
```

@jelhan looks like we should'n loosen Node.js minimum version?

---

Thanks for reviewing that closely. That was not intended.

```suggestion
    "node": "18.* || 20.* || >= 22"
```